dist/static/site.js

```diff
@@ -115,150 +63,122 @@ function setVolume(volume) {
 }
 
 function wsInit() {
-  document.getElementById('starttext').style.display = 'block';
-  document.getElementById('nameentry').style.display = 'none';
-  document.getElementById('interface').style.display = 'none';
+  // Reset styles
+  document.getElementById('starttext').hidden = false;
+  document.getElementById('nameentry').hidden = true;
+  document.getElementById('interface').hidden = true;
 
   // generate a relative websocket path
   const protocol = window.location.protocol === 'https:' ? 'wss://' : 'ws://';
   let defaultPath = window.location.pathname;
-  if (!/\/$/.test(defaultPath)) {
-    defaultPath += '/';
-  }
+  if (!/\/$/.test(defaultPath)) { defaultPath += '/'; }
   defaultPath += 'ws';
   sock = new WebSocket(protocol + window.location.host + defaultPath);
 
   sock.onopen = () => {
-    document.getElementById('starttext').style.display = 'none';
-    document.getElementById('lostconn').style.display = 'none';
-    const savedName = localStorage.getItem('jukebox_name');
+    document.getElementById('starttext').hidden = true;
+    document.getElementById('lostconn').hidden = true;
+    const savedName = localStorage.getItem('jukeboxName');
     if (savedName) {
       sendName(savedName);
     } else {
-      document.getElementById('nameentry').style.display = 'block';
+      document.getElementById('nameentry').hidden = false;
     }
   };
 
   sock.onmessage = (event) => {
     if (event.data === 'ok') { // Valid name
-      document.getElementById('nameentry').style.display = 'none';
-      document.getElementById('interface').style.display = 'block';
+      document.getElementById('nameentry').hidden = true;
+      document.getElementById('interface').hidden = false;
       document.getElementById('displayname').innerText = name;
-      localStorage.setItem('jukebox_name', name);
+      localStorage.setItem('jukeboxName', name);
       sock.onmessage = socketHandler;
     } else if (event.data === 'pong') {
-      isAlive = true;
     } else { // Error
-      document.getElementById('starttext').style.display = 'none';
-      document.getElementById('nameentry').style.display = 'block';
       document.getElementById('nameerror').innerText = event.data;
-      document.getElementById('nameerror').style.display = 'inline';
+      document.getElementById('nameerror').hidden = false;
     }
+    isAlive = true;
   };
 
-  sock.onclose = (event) => {
-    document.getElementById('lostconn').style.display = 'block';
-  };
+  sock.onclose = () => { document.getElementById('lostconn').hidden = false; };
 
-  sock.onerror = (event) => {
-    console.error(event);
-  };
+  sock.onerror = (event) => { console.error(event); };
 }
 
 function heartbeat() {
-  if (!sock || sock.readyState === 3) {
-    wsInit();
-  }
+  if (!sock || sock.readyState === 3) { wsInit(); }
   isAlive = false;
   sock.send('ping');
   setTimeout(checkIsAlive, 2000);
 }
 
+window.removeSong = (queuePos) => { sock.send(`remove ${queuePos}`); };
```

Comment:
```javascript
window.removeSong = queuePos => sock.send(`remove ${queuePos}`);
```

Reply:
Done, but I'll keep the brackets around the argument